packages/ui-components/src/components/Map/index.js

```diff
@@ -3,13 +3,14 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  *
  */
+export { Polygon } from 'react-leaflet';
```

Comment:
Unless I'm missing something, we can delete either this line or the Polygon.js file. I don't mind which one goes.

Reply:
Oh so this was weird...I tried importing `Polygon` from `react-leaflet` directly inside lesmis and everything broke! I could only find one other example of that being done and it turned out the file wasn't being used (which is why I deleted it)

Reply:
Again, something I could put more time into, but it also seemed kinda nice to not have lesmis directly import `react-leaflet` when it's getting all the other map stuff from `ui-components`

Reply:
I agree that's a good call keeping leaflet out of lesmis (for now anyway) and thanks for deleting that file. I still think that `ui-components/Map/Polygon.js` could be deleted right?

Reply:
Oh shoot I left that in huh...I see what you mean now! Sorry